### DISCUSSION TITLE
绳网道德规范

### DISCUSSION THREAD

**Discussion by Genshin-Neuvillette** [share121/inter-knot#2265] [其他]
![微信图片_20240815210051](https://github.com/user-attachments/assets/774d649b-c4c8-41cb-afc3-ac30243b5c33)
这是没有不明生物的社区，是玩家们安心讨论的社区。这不仅是绳匠的社区，也是各种玩家的社区。维护绳网环境，不仅需要我们维护社区环境，也需要对社区不良风气说不。

**Comment by Genshin-Neuvillette** [comment]
欢迎在此贴写下你对绳网的希望

**Comment by yileinafs** [comment]
希望可以越做越好<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by 369852147abc** [comment]
青衣不歪

**Comment by IHYAHED** [comment]
青衣不歪
